lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -0,0 +1,33 @@
+<div class="create-team wrapper-msg is-incontext urgency-low warning is-hidden">
+    <div class="msg">
+        <div class="msg-content">
+            <h3 class="title"><%- gettext("Oops!") %></h3>
```

Comment:
We need a heading here. The error message needs to be accessible in a list, out of context, so screen reader users can skip to the field in question, and then skip back to the error (if there are multiple incorrect fields). It just needs to be descriptive.